images/win/scripts/Tests/Git.Tests.ps1

```diff
@@ -1,5 +1,5 @@
 Describe "Git" {
-    $gitTools = 'bash', 'awk', 'git', 'git-lfs'
+    $gitTools = 'git', 'git-lfs'

```

Comment:
Could you please add additional tests to validate:
1) default bash / sh come from MSYS
2) shell folder exists and contains all necessary symlinks
3) both shell folder and MSYS folder are in PATH and MSYS folder located after System32

Reply:
Done.